src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Log export success when exporting telemetry using `http/protobuf`.
```

Comment:
We don't need to include this update in the changelog, as it doesn't have any relevance for the customer.

Reply:
Including these kinds of updates in the changelog could quickly become too noisy. Since this isn’t something most customers will act on directly, I’d lean toward leaving it out and this is what earlier maintainers too followed as guidelines in this repo.